cloudify_hostpool/rest/service.py

```diff
@@ -0,0 +1,67 @@
+########
```

Comment:
Yes, but I understood that you'd do it with name change in setup, I was confused, I'm changing it right now